site/content/3.13/data-science/arangographml/arangograph-ml.md

```diff
@@ -0,0 +1,183 @@
+- **Batch Size:** The nummer of documents processed in a single training iteration.  (Example, 256)
+- **Data Load Batch Size:** The number of documents loaded from ArangoDB into memory in a single batch during the data loading phase. (Example, 50000)
+- **Data Load Parallelism:** The number of parallel processes used when loading data from ArangoDB into memory for trainnig. (Example, 10)
+
+After setting these values, click the **Begin training** button to start the job.
+
+![Node Classification](../../../images/ml-nodeclassification.png)
+
+####  Node Embedding
+
+Node Embedding is used to generate vector embeddings (dense numerical representations) of graph nodes that capture structural and feature-based information.
+
+**Use cases include:**
+- Similarity search (Example, finding similar products, users, or documents)
+- Link prediction (Example, suggesting new connections)
+
+**Configuration Parameters:**
+- **Type of Training Job:** Node embeddings
+- **Target Vertex Collection:** Select the collection to generate embeddings for (Example, `movie` or `person`)
+- No label is required for training in this mode
+
+Once the configuration is complete, click **Begin training** to launch the embedding job.
+
+![Node Embeddings](../../../images/ml-node-embedding.png)
+
+
+After training is complete, the next step in the ArangoGraphML workflow is **Model Selection**.
+
+## Model Selection Phase
+
+Once the training is finished, the job status updates to READY FOR MODEL SELECTION. This means the model has been trained using the provided vertex and edge data and is now ready for evaluation.
+
+**Understanding Vertex Collections:**
+
+**X Vertex Collection:** These are the source nodes used during training. They represent the full set of nodes on which features were computed (Example, person, movie).
+
+**Y Vertex Collection:** These are the target nodes that contain labeled data. The labels in this collection are used to supervise the training process and are the basis for evaluating prediction quality.
+
+The target collection is where the model’s predictions will be stored once prediction is executed.
+
+**Model Selection Interface:**
+
+A list of trained models is displayed, along with performance metrics such as accuracy, Precision, Recall, F1 score, Loss.            
+Review the results of different model runs and configurations.
+
+Select the best performing model suitable for your prediction task.
+
+![Model Selection](../../../images/graph-ml-model.png)
+
+## Prediction Phase
+
+Once the best-performing model has been selected, the final step of the GraphML pipeline is to generate predictions for new or unlabeled data
+
+### Overview
+
+The Prediction interface allows inference to be run using the selected model. It enables configuration of how predictions are executed, which collections are involved, and whether new or outdated documents should be automatically featurized before prediction.
```

Comment:
Should add a statement about effects on quality when featurizing new/outdated docs